README.md

```diff
@@ -1,24 +1,69 @@
-# README
+# Bcycom Back End
+> This is the backend for the [Bcycom]() web application and below is what you can following to set up this web application...
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+### Deployment
 
-Things you may want to cover:
+[Base url]()
```

Comment:
- [ ] The link is not working pleases add the link to the API endpoint http://sleepy-forest-22507.herokuapp.com/